db/queries/users.js

```diff
@@ -0,0 +1,10 @@
+const db = require('../connection');
+
+const getUsers = () => {
```

Comment:
Could this be refactored to:

```javascript
const getUsers = async function() {
    const { rows: users } = await db.query("SELECT * FROM users;");
    return users;
};
```

Reply:
I remember you saying we don't teach `async` or `await`, so maybe what you have makes more sense :man_shrugging: